### DISCUSSION TITLE
"Emergency" message about seq version in CI logs

### DISCUSSION THREAD

**Discussion by alphapapa** [magit/magit#5146] [General]
Hi @tarsius,

I noticed messages like [this](https://github.com/alphapapa/ement.el/actions/runs/8708227081/job/23885080634#step:6:188) appearing in some of my CI linting logs.  Of course, I understand why you added that message, but in these cases, it's not relevant (no Magit code is being evaluated, and it doesn't appear to affect compilation), and it's making the logs quite noisy.  Would it be possible to prevent these messages from being output in these circumstances?  

I know, I'm asking to have our cake and eat it too, and I know this isn't really your fault.  But I thought that it won't do too much harm to ask about it.  :)

Thanks,
Adam

**Comment by tarsius** [comment]
> Would it be possible to prevent these messages from being output in these circumstances?

We could check if `GITHUB_ACTIONS`[^1] is `true` and not do it then.

I am not sure that would be the right thing to do though.

> in these cases [the message is] not relevant (no Magit code is being evaluated, and it doesn't appear to affect compilation),

Your ci/test does load and/or compile "Magit". If it doesn't depend on Magit, then why does it do that? If it does depend on Magit, then why doesn't it respect the dependencies that were specified for Magit?

If I suppress the warning to make you happy, this could make someone else unhappy who would have appreciated a warning about something in their code/setup not being quite right. Doing this would robe them of the opportunity to notice the issue, which in their case may be more consequential than you say it is in your case.




[^1]: Some others listed at https://stackoverflow.com/questions/73973332.
